### DISCUSSION TITLE
请教Spinner下拉框不显示问题（已解决）

### DISCUSSION THREAD

**Discussion by 1980922803** [xuexiangjys/XUI#138] [General]
最近用到MaterialSpinner  后台获取的list长度有40+，最外层布局是用的scrollView ,spinner.setItems(list)也有值，就是界面上点击下拉框下拉框不显示 ，但是点击不同区域会选到其他的值   
<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    xmlns:app="http://schemas.android.com/apk/res-auto"
>
 <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="40dp"
                android:gravity="center_vertical"
                android:orientation="horizontal">

                <TextView
                    android:layout_width="wrap_content"
                    android:layout_height="wrap_content"
                    android:layout_marginStart="0dp"
                    android:text="@string/app_ToArea" />

                <com.xuexiang.xui.widget.spinner.materialspinner.MaterialSpinner
                    android:id="@+id/spinner_stock_in_toArea"
                    style="@style/Material.SpinnerStyle"
                    app:ms_dropdown_height="100dp"
                    android:layout_width="280dp"
                    android:layout_marginStart="8dp"
                    android:layout_marginTop="2dp" />

            </LinearLayout>

        </LinearLayout>
</ScrollView >

**Comment by 1980922803** [comment]
界面Spinner.setDropdownMaxHeight

**Reply by xuexiangjys** [reply]
ms_dropdown_max_height
